bifromq-mqtt/bifromq-mqtt-server/src/main/java/com/baidu/bifromq/mqtt/service/LocalSessionBrokerService.java

```diff
@@ -46,6 +57,83 @@ public StreamObserver<WriteRequest> write(StreamObserver<WriteReply> responseObs
         return new LocalSessionWritePipeline(transientSessionMap, responseObserver);
     }
 
+    @Override
+    public void sub(SubRequest request, StreamObserver<SubReply> responseObserver) {
+        response(tenantId -> {
+            if (!transientSessionMap.containsKey(request.getInboxId())) {
+                return CompletableFuture.completedFuture(SubReply.newBuilder()
+                        .setReqId(request.getReqId())
+                        .setResult(SubReply.Result.NO_INBOX)
+                        .build());
+            }else {
+                MQTT3TransientSessionHandler session = (MQTT3TransientSessionHandler) transientSessionMap
```

Comment:
1. why repeat sub/unsub logic here?
2. sub/unsub must be run in session's event loop to avoid race condition